app/lib/activity_stream_reader.rb

```diff
@@ -115,18 +115,13 @@ def refresh_updated_items(parent_objects_for_update)
     parent_objects_for_update.each do |parent_object_array|
       oid = parent_object_array[0]
       metadata_source = parent_object_array[1]
-      metadata_cloud_url = MetadataCloudService.build_metadata_cloud_url(oid, metadata_source)
-      full_response = MetadataCloudService.mc_get(metadata_cloud_url)
-      MetadataCloudService.save_mc_json_to_file(full_response, oid, metadata_source)
       po = ParentObject.find_by(oid: oid)
-      if metadata_source == "ladybird"
-        po.last_ladybird_update = DateTime.current
-      elsif metadata_source == "ils"
-        po.last_voyager_update = DateTime.current
-      elsif metadata_source == "aspace"
-        po.last_aspace_update = DateTime.current
+      if po
```

Comment:
Non-blocking question: is getting all of the records from the MetadataCloud the behavior we want here in the end? 

I'm having a little trouble knowing how much parallelism we'll want in the end wrt having non-authoritative metadata source records accessible via the Management application. 

I'm thinking as we scale up, perhaps we want to default to only caring about dependent objects if they relate to the authoritative metadata source?

Reply:
Great question and one I think we need to pull out of here and discuss with Mark and Mike